### PR TITLE
'test'

### PR DESCRIPTION
**Pull request by chenyongchangg**

### CODE REVIEW

**Comment by chenyongchangg** [comment]
sdf

**Comment by chenyongchangg** [comment]
dsf

**Comment by chenyongchangg** [comment]
dsf

File: src/components/SecondaryMarket/component/Details/Details.vue [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    import {marketFetch, yibanAuth} from "../../config/fetchUtil"
+
+
+    export default {
+        name: 'index',
+        mixins: [updateData, loading(marketFetch), fetchVq(yibanAuth)],
+
+        mounted() {
+            marketFetch.getJsonData("/secondhand/browse/onearticle", {
+                'articleid': 1
```
Comment (1409070209) [inline src/components/SecondaryMarket/component/Details/Details.vue:51]: use articleId

File: src/components/SecondaryMarket/component/Details/Details.vue [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    </div>
+</template>
+
+<script>
+
+
+    import HeaderSection from '../../../../common/components/HeaderSection'
+    import ManagerHeader from '../../common-component/ManagerHeader.vue'
+    import MangerDetails from '../../common-component/ManagerDetails.vue'
+    import coll2 from '../../media/coll2.png'
```
Comment (1409070209) [inline src/components/SecondaryMarket/component/Details/Details.vue:35]: user require function

**Pull request merged**